test/e2e/network/endpointslice.go

```diff
@@ -126,9 +126,10 @@ var _ = SIGDescribe("EndpointSlice", func() {
 		}
 
 		err := cs.CoreV1().Services(svc.Namespace).Delete(context.TODO(), svc.Name, metav1.DeleteOptions{})
-		framework.ExpectNoError(err)
+		framework.ExpectNoError(err, "error deleting Service")
 
-		// Expect Endpoints resource to be deleted when Service is.
+		// Expect Endpoints resource to not be deleted when Service is.
+		// TODO(robscott): Update this test when this bug is fixed.
```

Comment:
If we expect that the `Endpoints` resource will not be deleted here wouldn't this test always fail?

Reply:
yep, I don't understand the comment either, the code seems to check they are deleted

Reply:
🤦 Apparently I momentarily confused this part with another part that is waiting for an Endpoints related fix and thought I'd forgotten to add a TODO here. As it turns out the TODO already existed in the appropriate place: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/endpointslice.go#L296-L297. That bug will actually be fixed with an [open PR](https://github.com/kubernetes/kubernetes/pull/93030) so I need to remember to connect those dots.